code/items/base/Ammo.cs

```diff
@@ -15,13 +17,23 @@ public enum AmmoType : byte
 	Rifle
 }
 
+
+
 [Hammer.Skip]
 public abstract partial class Ammo : Prop, IEntityHint, IUse
 {
-	[Net]
-	public int CurrentCount { get; set; }
 	public virtual AmmoType Type => AmmoType.None;
+
+	///<summary>How much ammo is left in this box.</summary>
+	[Net] public int CurrentCount { get; set; }
```

Comment:
Keep `[Net]` values at the top of the class, and make `[Net]` on a separate line.